src/EnergyPlus/SetPointManager.cc

```diff
@@ -8111,6 +8473,64 @@ void DefineIdealCondEntSetPointManager::SetupMeteredVarsForSetPt(EnergyPlusData
     this->CndPumpVarIndex = VarIndexes(1);
 }
 
+void DefineSysNodeResetSetPointManager::calculate(EnergyPlusData &state)
+{
+    // SUBROUTINE INFORMATION:
+    //       AUTHOR         Wooyoung Jung, Jeremy Lerond, and Jian Zhang, PNNL
+    //       DATE WRITTEN   Jan 2022
+    //       MODIFIED       na
+    //       RE-ENGINEERED  na
+
+    // PURPOSE OF THIS SUBROUTINE:
+    // Set a temperature setpoint at a user-defiend node based on the temperature at another user-specified (i.e., reference) node.
+    // The setpoint is determined using a linear relationship defined by the user. In general, the higher the reference temperature,
+    // the lower the setpoint.
+
+    // REFERENCE:
+    // Guanghua Wei, W. Dan Turner, David E. Claridge, and Mingsheng Liu. 2003. Single-Duct Constant Air Volume System Supply Air Temperature Reset:
+    // Using Return Air Temperature or Outside Air Temperature? Architectural Engineering Conference (AEI) 2003. Sep 17-20, Austin, TX.
+
+    // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
```

Comment:
Purpose and Reference above are fine, but headings like this aren't used anymore. And we try to just declare variables when first used rather than at the top of the function.

Reply:
Sorry, I told him to leave that in. Didn't know what the new template was.

Reply:
No problem, there isn't really a new template, per se. It's a minimalist approach.

Reply:
This may be the first time that EnergyPlus and minimalist have appeared in the same sentence.